src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java

```diff
@@ -823,6 +823,36 @@ public void testLoggedTime() throws IOException {
 
     }
 
+    @Test
+    public void testLoggedTimeInProcessFiles() throws IOException {
+        final CheckstyleAntTaskLogStub antTask = new CheckstyleAntTaskLogStub();
+        antTask.setConfig(getPath(CONFIG_FILE));
+        antTask.setProject(new Project());
+        antTask.setFile(new File(getPath(FLAWLESS_INPUT)));
+        final long startTime = System.currentTimeMillis();
+        antTask.execute();
+        final long endTime = System.currentTimeMillis();
+        final long overallLoggedTime = endTime - startTime;
```

Comment:
let's rename `overallLoggedTime` to `testingTime` or `executingTime` because it's not connected with logging.
(and overallLoggedTime parameter in assertLoggedTime too)

Reply:
Done